gridmenuview/src/androidTest/java/com/bcc/gridmenuview/MenuItemTest.java

```diff
@@ -0,0 +1,43 @@
+package com.bcc.gridmenuview;
+import android.graphics.drawable.Drawable;
+import androidx.test.platform.app.InstrumentationRegistry;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class MenuItemTest  {
+
+    private MenuItem  menuItem;
+
+    @Before
+    public void setUp()  {
+        menuItem = new MenuItem("Satu",InstrumentationRegistry
+                .getInstrumentation()
+                .getContext()
+                .getDrawable(R.drawable.ic_launcher_background));
+
+    }
+
+    @Test
+    public void testMenuItemNotNull(){
```

Comment:
don't use the test prefix in the method name for the test case because the @Test annotations already represent that it is a test case